subsys/bluetooth/mesh/shell.c

```diff
@@ -2603,6 +2676,8 @@ SHELL_STATIC_SUBCMD_SET_CREATE(mesh_cmds,
 		      0),
 	SHELL_CMD_ARG(app-key-add, NULL, "<NetKeyIndex> <AppKeyIndex> [val]",
 		      cmd_app_key_add, 3, 1),
+	SHELL_CMD_ARG(app-key-del, NULL, "<NetKeyIndex> <AppKeyIndex>",
+		      cmd_app_key_del, 2, 0),
```

Comment:
Have to include the `mesh` parameter in the count here, so the argument count parameter should be `3` here. The `mod-app-unbind` entry is correct.